README

```diff
@@ -274,3 +274,68 @@ refresh			Attempts to reinsert all modules that are currently
 xml			Build a policy.xml from the XML included with the
 			base policy headers and any XML in the modules in
 			the current directory.
+
+
+## How to work with SELinux
+### Introduction
+
+To be able to add and change the SELinux policy, you need to clone the repository with refpolicy, and make changes with a separate commit.
+
+Each process or resource (file, dir etc.) has a label (type) that describes what can be accessed.
+
+SELinux considers subject-access-object rule set. On a Linux system, subjects are processes, and object is the resource on which an action applies.
+
+Refpolicy contains the following file types:
+1. **te** - the file contains rules describing what the domain can do (labels of the subjects or objects). This file does not say that this process can run another file, it just says that the process labeled A can read the file labeled B.
+2. **fc** - describes the security context (labels) on a file, directory, etc., which will be applied when the policy is install. By default, directories or files inherit the context of their parents, but you can manually set the desired context through this file
```

Comment:
but the desired context could be manually set using this file

Reply:
Done